### PR TITLE
Randomprime 1.18.1 + Cutscene Migration

### PR DESCRIPTION
**Pull request by toasterparty**
image

### CODE REVIEW

File: randovania/games/prime1/layout/preset_describer.py [hunk 1]
```diff
@@ -17,8 +17,8 @@
     LayoutCutsceneMode.MAJOR: "Major cutscene removal",
     LayoutCutsceneMode.MINOR: "Minor cutscene removal",
     LayoutCutsceneMode.COMPETITIVE: "Competitive cutscene removal",
-    LayoutCutsceneMode.SKIPPABLE: "Skippable cutscenes (Experimental)",
-    LayoutCutsceneMode.SKIPPABLE_COMPETITIVE: "Competitive cutscene removal (Experimental)",
+    LayoutCutsceneMode.SKIPPABLE: "Skippable cutscenes",
```
Comment (henriquegemignani) [inline randovania/games/prime1/layout/preset_describer.py:20]: We could go with skippable cutscenes not being mentioned, but vanilla mentions they're not.

File: randovania/layout/preset_migration.py [hunk 2]
```diff
@@ -977,6 +977,16 @@ def _migrate_v62(preset: dict) -> dict:
     return preset
 
 
+def _migrate_v63(preset: dict) -> dict:
+    if preset["game"] == "prime1":
+        if preset["configuration"]["qol_cutscenes"] in ["original", "skippable"]:
+            preset["configuration"]["qol_cutscenes"] = "skippable"
+        else:
+            preset["configuration"]["qol_cutscenes"] = "skippablecompetitive"
```
Comment (henriquegemignani) [inline randovania/layout/preset_migration.py:985]: Is this the enum name? ugh. Could've been `skippable-competitive` but I see this is an existing value.

Comment (toasterparty) [reply randovania/layout/preset_migration.py:985]: This value is used by the patcher. Ideally it's `SkippableCompetitive` but I didn't want to rename existing to make consistent

Comment (toasterparty) [reply randovania/layout/preset_migration.py:985]: At a later date, I will have a PR that updates everything used by the prime exporter to conform to the new schema

Comment (toasterparty) [reply randovania/layout/preset_migration.py:985]: I'll make that separation in the schema conformity PR then

**Pull request merged**